src/WebJobs.Extensions.DurableTask/DurableOrchestrationClient.cs

```diff
@@ -42,12 +42,26 @@ public class DurableOrchestrationClient
         /// </summary>
         /// <param name="request">The HTTP request that triggered the current function.</param>
         /// <param name="instanceId">The unique ID of the instance to check.</param>
-        /// <returns>An HTTP response which may include a 202 and locaton header.</returns>
+        /// <returns>An HTTP response which may include a 202 and location header.</returns>
         public HttpResponseMessage CreateCheckStatusResponse(HttpRequestMessage request, string instanceId)
         {
             return this.config.CreateCheckStatusResponse(request, instanceId, this.attribute);
         }
 
+        /// <summary>
+        /// Creates an HTTP response for checking the status of the specified instance supporting synchronous response as well. 
+        /// </summary>
+        /// <param name="request">The HTTP request that triggered the current function.</param>
+        /// <param name="instanceId">The unique ID of the instance to check.</param>
+        /// <param name="timeout ">Total allowed timeout for output from the durable function.</param>
+        /// <param name="timeout">Total allowed timeout for output from the durable function.</param>
+        /// <param name="retryInterval">Timeout between checks for output from the durable function.</param>
```

Comment:
Let's add a note saying that the default is 5 seconds.

Reply:
Sure.